kubernetes-ingress/templates/default-backend-deployment.yaml

```diff
@@ -54,6 +54,13 @@ spec:
             - name: http
               containerPort: {{ .Values.defaultBackend.containerPort }}
               protocol: TCP
+          {{- if .Values.defaultBackend.extraEnvs }}
```

Comment:
Can you please move first `{{- end }}` below second `{{- end }}` so that `{{- if .Values.defaultBackend.extraEnvs }}` covers both the test if we have extraEnvs and iteration through `.Values.defaultBackend.extraEnvs`?